### DISCUSSION TITLE
Hyperlinks

### DISCUSSION THREAD

**Discussion by sunjam** [selfhst/icons#149] [Report Issue / Error]
I noticed each icon, which is very nice, does not include a hyperlink to the related project within either the name or the icon.  I can imagine people using the icon list to discover projects.  What do you think of:
- adding hyperlinks to project repos by name
  - [Nextcloud](https://github.com/nextcloud/server)
- or by adding hyperlink to the rendered icon

**Comment by shollyethan** [comment]
Hi. I'd certainly be up for it if there was enough demand, but [I already have a directory of software](https://selfh.st/apps) built specifically for self-hosted software discovery. I'd like to keep this collection focused purely on icons, if possible.

**Reply by sunjam** [reply]
I see. Links are on the apps page, but not on the icons page.
